quarterly/2023-04-25-consensus.md

```diff
@@ -79,6 +79,22 @@ hide_table_of_contents: false
     - System-level tests not passing.
     - System-level benchmarks showing poor performance.
 
-#### Fostering collaboration 
+#### Support
+
+- Design Consensus side of hardfork-enactment in the Voltaire phase
+  ([#4180](https://github.com/input-output-hk/ouroboros-network/issues/4180)).
+- TODO Anything we need to plan for new crypto integration?
+
+#### Tech debt
+
+- Check absence of thunks during nightly CI tests
+  ([#3001](https://github.com/input-output-hk/ouroboros-network/issues/3001) and
+  [#4006](https://github.com/input-output-hk/ouroboros-network/issues/4006))
+- Keep backwards compatibility with previous ledger snapshot versions
+  [#4434](https://github.com/input-output-hk/ouroboros-network/issues/4434).
+- Estimate the number of file descriptors Consensus needs
```

Comment:
This is currently filed under Support, see https://github.com/input-output-hk/ouroboros-consensus/issues/20.

Reply:
Done, thanks!